specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/examples/amlFilesystems_Get.json

```diff
@@ -0,0 +1,65 @@
+{
+  "parameters": {
+    "resourceGroupName": "scgroup",
+    "amlFilesystemName": "fs1",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "api-version": "2020-07-23-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "tags": {
+          "Dept": "ContosoAds"
+        },
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/scgroup/providers/Microsoft.StorageCache/amlFilesystems/fs1",
+        "location": "westus",
+        "name": "fs1",
+        "type": "Microsoft.StorageCache/amlFilesystem",
+        "properties": {
+          "storageCapacityTiB": 3072,
+          "availabilityZones": ["1", "2"],
+          "health": {
+            "state": "Healthy",
+            "statusDescription": "amlFilesystem is ok."
+          },
+          "provisioningState": "Succeeded",
+          "subnet": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/scgroup/providers/Microsoft.Network/virtualNetworks/amlvnet/subnets/sub1",
+          "encryptionSettings": {
+            "keyEncryptionKey": {
+              "keyUrl": "https://examplekv.vault.azure.net/keys/kvk/3540a47df75541378d3518c6a4bdf5af",
+              "sourceVault": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/some-rg/providers/Microsoft.KeyVault/vaults/examplekv"
+            }
+          },
+          "maintenanceWindow" : "2020-09-13T13:15Z",
+          "maintenanceStatus": {
+            "currentFirmwareVersion": "5.3.23",
+            "firmwareUpdateStatus": "available",
```

Comment:
Maintenance window could update either Node OS or Node lustre version. Should have currentversion/status/pendingupdate for both OS and Lustre versions.

Reply:
Does the customer care about the difference?   I would think that a customer of a managed product like this would prefer a single version number that encompassed the entirely of software running on the cluster (eg: os + lustre).

Reply:
Customer facing version can be a single version, but the use-case here is to apply some security patch so we need a way internally to map the customer facing version to a specific version+patches  for OS and Lustre, especially for long-running clusters that can have multiple maintenance patches.   But I agree that this detail need not be in the swagger API.